esmvalcore/cmor/_fixes/cmip6/cesm2_waccm.py

```diff
@@ -1,6 +1,42 @@
 """Fixes for CESM2-WACCM model."""
+import iris
+import numpy as np
+
 from .cesm2 import Tas as BaseTas
+from ..fix import Fix
 
 
 class Tas(BaseTas):
     """Fixes for tas."""
+
+
+class Ua(Fix):
+    """Fixes for ua."""
+
+    def fix_metadata(self, cubes):
+        """Fix non-monotonic time axis.
+
+        Parameters
+        ----------
+        cube : iris.cube.CubeList
+
+        Returns
+        -------
+        iris.cube.Cube
+
+        """
+        cube = self.get_cube_from_list(cubes)
+        dim_coords = [coord.name() for coord in cube.coords(dim_coords=True)]
+        if 'time' in dim_coords:
+            return cubes
+        coord = cube.coord('time')
+        idx_sorted = np.argsort(coord.points)
+        coord.points = coord.points[idx_sorted]
+        coord.bounds = None
+        # coord.guess_bounds('time')  # fails for coordinate 'time'
```

Comment:
What is failing exactly?

Reply:
Agree with @bouweandela question. @hb326, note that `coord` already refers to time, so I am not sure why there is another mention of time in the args. Certainly `guess_bounds` works with time coordinates in general.